Library/Homebrew/brew.sh

```diff
@@ -764,7 +764,7 @@ then
   export HOMEBREW_DEVELOPER_MODE="1"
 fi
 
-if [[ -n "${HOMEBREW_INSTALL_FROM_API}" && -n "${HOMEBREW_DEVELOPER_COMMAND}" ]]
+if [[ -n "${HOMEBREW_INSTALL_FROM_API}" && -n "${HOMEBREW_DEVELOPER_COMMAND}" && "${HOMEBREW_COMMAND}" != "irb" ]]
```

Comment:
👍🏻 for now. I'm thinking we may want to have a longer list of commands we allow here.

Reply:
I already have a PR in the works for this